### ISSUE TITLE
Traceback if photo.original_filename is None

### ISSUE THREAD

**Issue by jstrine** [RhetTbull/osxphotos#267]
In my library, several photos seemed to not have an "original_filename".  As such, those photos would case a traceback and stop the export.

Photo library is from macOS 10.13, but has been upgraded over many, many years, so it may have been legacy entries through the years that were causing issues.

```
Exporting 4769 photos to photos...
Traceback (most recent call last):
  File "osxphotos/cli.py", line 19, in <module>
    cli()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/decorators.py", line 33, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/jstrine/Desktop/osxphotos/osxphotos/__main__.py", line 1776, in export
    results = export_photo(
  File "/Users/jstrine/Desktop/osxphotos/osxphotos/__main__.py", line 2437, in export_photo
    export_results = photo.export2(
  File "/Users/jstrine/Desktop/osxphotos/osxphotos/photoinfo/_photoinfo_export.py", line 455, in export2
    fname = pathlib.Path(fname)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/pathlib.py", line 1038, in __new__
    self = cls._from_parts(args, init=False)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/pathlib.py", line 679, in _from_parts
    drv, root, parts = self._parse_args(args)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/pathlib.py", line 663, in _parse_args
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

**Comment by RhetTbull** [comment]
Interesting -- I've never run across. a case where original filename was null so obviously didn't check for this case.  Thanks for the PR -- I'll merge it as soon as I have a chance to look at the changes.  It might make more sense to set original filename to filename in PhotoInfo if it's null:

https://github.com/RhetTbull/osxphotos/blob/e951e5361e59060229787bb1ea3fc4e088ffff99/osxphotos/photoinfo/photoinfo.py#L84-L96

This would fix this issue for other uses besides just export (e.g. for use in the template system).

**Comment by jstrine** [comment]
I'll look at the above (updating PhotoInfo) and rerun an export on my library.  I agree that's probably a more elegant solution.  It'll take me a few days, but I'll let you know what I find.  Thanks!

**Comment by RhetTbull** [comment]
I updated PhotoInfo.original_filename to use PhotoInfo.filename if original_filename is null and added a test for missing this condition.

**Comment by jstrine** [comment]
I ran two tests today to ease some curiosity.

Using v0.36.23, the export of my library worked without issue.
I then reverted commit 5352aec (every else the same), and the export worked without issue.

Obviously the underlying fixes you made in fa33218 have solved it for sure and my "workaround" for the export isn't needed per se.

Thanks for making this awesome utility!
